src/hooks/ContributorProposal/useSubmitProposal.ts

```diff
@@ -0,0 +1,164 @@
+import { useState } from 'react';
+import BigNumber from 'bignumber.js';

```

Comment:
Question: I have seen sometimes we import this from diff places. `utils/bignumber` | `bignumber.js` | `ethers`. Not sure if there is one that is correct but by context I understand the idea of having it on utils is to use it from there. Let me know if makes sense or not.